nucleus/dataset.py

```diff
@@ -525,3 +531,134 @@ def get_scene(self, reference_id) -> Scene:
                 requests_command=requests.get,
             )
         )
+
+    def export_predictions(self, model):
+        """Exports all predications from a model run"""
+        json_response = self._client.make_request(
+            payload=None,
+            route=f"dataset/{self.id}/model/{model.id}/export",
+            requests_command=requests.get,
+        )
+        return format_prediction_response({ANNOTATIONS_KEY: json_response})
+
+    def calculate_evaluation_metrics(self, model, options=None):
+        """
+        class_agnostic -- A flag to specify if matching algorithm should be class-agnostic or not.
+                          Default value: True
+
+        allowed_label_matches -- An optional list of AllowedMatch objects to specify allowed matches
+                                 for ground truth and model predictions.
+                                 If specified, 'class_agnostic' flag is assumed to be False
+
+        Type 'AllowedMatch':
+        {
+            ground_truth_label: string,       # A label for ground truth annotation.
+            model_prediction_label: string,   # A label for model prediction that can be matched with
+                                              # corresponding ground truth label.
+        }
+
+        payload:
+        {
+            "class_agnostic": boolean,
+            "allowed_label_matches": List[AllowedMatch],
+        }"""
+        if options is None:
+            options = {}
+        return self._client.make_request(
+            payload=options,
+            route=f"dataset/{self.id}/model/{model.id}/calculateEvaluationMetrics",
+        )
+
+    def upload_predictions(
```

Comment:
I think there should be parity in naming between predictions and annotations.  If this func is named `upload_predictions` maybe `annotate` should be `upload_annotations`?  Or vice versa, `predict`, `annotate`

Reply:
I wanted to differentiate between the eventual

model.predict

and 

dataset.upload_predictions

To be clear that actual prediction is happening in one case and in another prediction is actually running on hosted model inference.